src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/Reference.cs

```diff
@@ -38,7 +38,7 @@ public Reference()
             _refTarget = null;
             _refTargetType = ReferenceTargetType.UriReference;
             _cachedXml = null;
-            _digestMethod = SignedXml.XmlDsigSHA1Url;
+            _digestMethod = SignedXml.XmlDsigSHA256Url;
```

Comment:
How about replacing this with an internal const for a default algorithm, so in the future replacement is easier?